src/libponyc/pass/docgen.c

```diff
@@ -796,7 +796,7 @@ static void doc_entity(docgen_t* docgen, docgen_opt_t* docgen_opt, ast_t* ast)
 
   // Now we can write the actual documentation for the entity
   fprintf(docgen->type_file, "# %s", name);
-  doc_type_params(docgen, docgen_opt, tparams, true);
+  doc_type_params(docgen, docgen_opt, tparams, true, true);
```

Comment:
we need a false here, since those type params are part of a markdown heading

Reply:
E.g. the title of the `collections.Range` docs is still *line-broken*

Reply:
You're right! This is a slightly different, but of course related, bug: I've just found an instance of it [in the Flag  docs](https://stdlib.ponylang.org/collections-Flag/).

Thanks, will update. 👍

Reply:
(Oh I should have updated my page -- didn't see your second comment when adding mine. Anyhow, you're right. It's updated.)